nengo/tests/conftest.py

```diff
@@ -232,6 +233,11 @@ def pytest_runtest_setup(item):
         if skip:
             pytest.skip(" and ".join(skipreasons))
 
+    if 'Simulator' in item.fixturenames:
+        for test, reason in _Simulator.unsupported:
+            if re.match(test, item.name):
```

Comment:
`item.name` is not necessarily unique. It might be better to use `item.nodeid`, but that adds a dependence on the actual filename because the strings look like `nengo/tests/test_ensemble.py::test_constant_scalar[nl0]` and they get weird for tests in classes: `nengo/tests/test_something.py::TestSomething::()::test_foo`